MantidQt/MantidWidgets/src/InstrumentView/RotationSurface.cpp

```diff
@@ -86,102 +86,111 @@ void RotationSurface::init() {
-                              "An exception was thrown. See log for detail.");
-                        }
-
-                        // find the overall edges in u and v coords
-                        findUVBounds();
-
-                        // apply a shift in u-coord either found automatically
-                        // or set manually
-                        if (!m_manual_u_correction) {
-                          // automatic gap correction
-                          findAndCorrectUGap();
-                        } else {
-                          // apply manually set shift
-                          m_u_min = manual_u_min;
-                          m_u_max = manual_u_max;
-                          for (size_t i = 0; i < m_unwrappedDetectors.size();
-                               ++i) {
-                            auto &udet = m_unwrappedDetectors[i];
-                            udet.u = applyUCorrection(udet.u);
-                          }
-                        }
-
-                        double dU = fabs(m_u_max - m_u_min);
-                        double dV = fabs(m_v_max - m_v_min);
-                        double du = dU * 0.05;
-                        double dv = dV * 0.05;
-                        if (m_width_max > du && std::isfinite(m_width_max)) {
-                          if (du > 0 && !(dU >= m_width_max)) {
-                            m_width_max = dU;
-                          }
-                          du = m_width_max;
-                        }
-                        if (m_height_max > dv && std::isfinite(m_height_max)) {
-                          if (dv > 0 && !(dV >= m_height_max)) {
-                            m_height_max = dV;
-                          }
-                          dv = m_height_max;
-                        }
-
-                        m_viewRect = RectF(QPointF(m_u_min - du, m_v_min - dv),
-                                           QPointF(m_u_max + du, m_v_max + dv));
+  PRAGMA_OMP(parallel for)
+  for (int ii = 0; ii < int(ndet); ++ii) {
+    if (!exceptionThrown)
+      try {
+        size_t i = size_t(ii);
+
+        unsigned char color[3];
+        Mantid::detid_t id = m_instrActor->getDetID(i);
+
+        boost::shared_ptr<const Mantid::Geometry::IDetector> det;
+        try {
+          det = inst->getDetector(id);
+        } catch (Mantid::Kernel::Exception::NotFoundError &) {
+        }
+
+        if (!det || detectorInfo.isMonitor(detectorInfo.indexOf(id)) ||
+            (id < 0)) {
+          // Not a detector or a monitor
+          // Make some blank, empty thing that won't draw
+          m_unwrappedDetectors[i] = UnwrappedDetector();
+        } else {
+          // A real detector.
+          m_instrActor->getColor(id).getUB3(&color[0]);
+
+          // Position, relative to origin
+          // Mantid::Kernel::V3D pos = det->getPos() -
+          // m_pos;
+          Mantid::Kernel::V3D pos = m_instrActor->getDetPos(i) - m_pos;
```

Comment:
As @AntonPiccardoSelg points out the code is really unchanged. Those changes are only to whitespace. Whitespace changes are also a separate commit so could be dropped easily.